index.html

```diff
@@ -193,8 +193,7 @@ <h1>bam2.iobio</h1>
                 <p>This is custom content for the Read Coverage Distribution histogram.</p>
               </div>
             </iobio-label-info-button>
-
-            <iobio-histogram broker-key="coverage_hist"></iobio-histogram>
+            <iobio-histogram broker-key="coverage_hist" custom-tooltip-format custom-xaxis-tickformat custom-yaxis-tickformat></iobio-histogram>
```

Comment:
This is fine for this one case, but what if the user wants to provide custom formatters?

Reply:
That's a great point! 

In that case, setting up the property would be a more effective solution, allowing the user to control it later on as needed. Thanks for guiding me to think it through more carefully.